script.js

```diff
@@ -0,0 +1,47 @@
+const navItems = document.querySelector('.nav-items');
+const navLinks = document.querySelectorAll('.nav-items a');
+const hamburgerBtn = document.querySelector('.hamburger');
+
+const closeBtn = document.createElement('img');
+closeBtn.src = 'img/toolbar.png';
```

Comment:
- [ ] [OPTIONAL] Your close button is not getting displayed correctly. I think you are using `toolbar.png` instead of `Toolbar.png`. Kindly update the path. 🤓️

```
closeBtn.src = 'img/Toolbar.png';
```